peep.py

```diff
@@ -362,9 +362,11 @@ def package_finder(argv):
     # Carry over PackageFinder kwargs that have [about] the same names as
     # options attr names:
     possible_options = [
-        'find_links', FORMAT_CONTROL_ARG, 'allow_external', 'allow_unverified',
```

Comment:
Oh! Right you are. I suppose people running their own indices could still be using the old protocol version, but they'd probably also have to be using an older version of pip, so they'd be welcome to continue using an old version of peep as well. Carry on.